### ISSUE TITLE
Do not send password as query

### ISSUE THREAD

**Issue by coughingmouse** [H-M-H/Weylus#31]
tl;dr: please remove password feature because it's making the app even more unsecure

First of all, THANK YOU so much for publishing this Amazing software for free!
It works well enough for me on my ancient ThinkPad T420 with Ubuntu 20.04 OS. (Although I haven't tested "use as second monitor" thingy)

However, it seems you've added in a feature that's in my opinion not really a feature but rather a security threat. You've added in a password feature using query in the URL. However, the UR location nor the address bar in modern browsers should be used for any secure information. I'm no expert but the least, URLs are stored in browser history which anyone using the computer can easily get access to, and the content in the address bar is sent to a search engine for autocomplete. This is especially unsecure if the user's browser is slightly compromised so that the autocomplete search engine is redirected onto a malicious server (which is the case for my parents' laptop currently). The password is also visible in the unencrypted packet of HTTP which this program uses.

As virtually everyone knows, HTTP is by nature unsecure and naming a feature named  "password" may give the user the impression of security while using this application. If the user thus uses their password which they use for other uses on Weylus, this could be another security threat.

As of now, I believe it's better for security to remove password feature as is. So I suggest you do. 

Again, THANK YOU SOOO MUCH for this masterpiece!

p.s. It would be nice if you could afford to add in a fullscreen feature.

**Comment by H-M-H** [comment]
> First of all, THANK YOU so much for publishing this Amazing software for free!

Thanks, I am happy you like it!

> password feature using query in the URL

That is true. The reason why it is encoded in a GET request is that it can be passed along with the QR-Code and you can easily create a bookmark, to make things as seamless as possible.

> The password is also visible in the unencrypted packet of HTTP which this program uses.

Weylus is supposed to be used via local network, so if you are afraid of MITM attacks you probably have other problems than getting Weylus to run. Additionally encrypting everything would lead to higher latency. (But maybe there is a use case that actually needs encryption I did not think about, feel free to tell me about it!)

> As virtually everyone knows, HTTP is by nature unsecure and naming a feature named "password" may give the user the impression of security while using this application. If the user thus uses their password which they use for other uses on Weylus, this could be another security threat.

You are right, calling this feature "password" can very well be interpreted in the wrong way. Nonetheless, I do not want to remove it as it does provide access control, which is required for say the campus network of some university. What do you think about renaming it, maybe "access code" and adding some explanatory text to the GUI?

> p.s. It would be nice if you could afford to add in a fullscreen feature.

Try adding the Weylus page to your home screen on your tablet, opening it via home screen should enable fullscreen.

**Comment by coughingmouse** [comment]
"(Optional) access code" sounds nice enough for me. Thanks for the feedback!

> Try adding the Weylus page to your home screen on your tablet, opening it via home screen should enable fullscreen.

Thanks again! Works like a charm. It would be nice if you added this advice in the readme as a tip for dumb people like me.

**Comment by H-M-H** [comment]
Fixed with https://github.com/H-M-H/Weylus/commit/4af00391f98662b47ecb2c3216d33f67a52c5685.

> It would be nice if you added this advice in the readme as a tip for dumb people like me.

It was already there (under Running) but somewhat easy to miss, it got its own section now.
